server/api/namespace.go

```diff
@@ -0,0 +1,59 @@
+package api
+
+import (
+	"context"
+
+	"github.com/odpf/stencil/server/namespace"
+	stencilv1 "github.com/odpf/stencil/server/odpf/stencil/v1"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func createNamespaceRequestToNamespace(r *stencilv1.CreateNamespaceRequest) namespace.Namespace {
+	return namespace.Namespace{
+		ID:          r.GetId(),
+		Format:      r.GetFormat().String(),
+		Description: r.GetDescription(),
+	}
+}
+
+func namespaceToProto(ns namespace.Namespace) *stencilv1.Namespace {
+	return &stencilv1.Namespace{
+		Id:          ns.ID,
+		Format:      stencilv1.Schema_Format(stencilv1.Schema_Format_value[ns.Format]),
```

Comment:
Why we are having Format at both places in namespace and schema APIs?

Reply:
Because user can override format and compatibility level at schema level, if format and compatibility is different at namespace level.